.pre-commit-config.yaml

```diff
@@ -24,7 +25,7 @@ repos:
         name: isort (python)
     -   id: isort
```

Comment:
The two `isort` hooks can be combined into one
```suggestion
    -   id: isort
        types: [text]
        types_or: [python, cython]
```

Reply:
Thank you @MarcoGorelli for the explanation.